src/table/TableHeadWrapper.jsx

```diff
@@ -65,22 +78,16 @@ function TableHeadWrapper({
                 )
               }
               onMouseDown={() => handleClickToFocusHead(columnIndex, rootElement, setfocusedCellCoord)}
+              onClick={() =>
+                !selectionsAPI.isModal() && !constraints.active && changeSortOrder(layout, column.isDim, columnIndex)
+              }
             >
-              <TableSortLabel
-                active={layout.qHyperCube.qEffectiveInterColumnSortOrder[0] === columnIndex}
-                direction={column.sortDirection}
-                onClick={() =>
-                  // when cells are selected or in edit mode, it should not be able to do the sorting
-                  !selectionsAPI.isModal() && !constraints.active && changeSortOrder(layout, column.isDim, columnIndex)
-                }
-                tabIndex={-1}
-              >
+              <TableSortLabel active={isCurrentColumnActive} direction={column.sortDirection} tabIndex={-1}>
                 {column.label}
-                {layout.qHyperCube.qEffectiveInterColumnSortOrder[0] === columnIndex ? (
-                  <span className={classes.visuallyHidden}>
-                    {column.sortDirection === 'desc' ? 'sorted descending' : 'sorted ascending'}
-                  </span>
-                ) : null}
+                <span className={classes.visuallyHidden} data-testid={`VHL-for-col-${columnIndex}`}>
+                  {isCurrentColumnActive && `${currentSortDir}. `}
+                  Press space or enter to sort on the column.
```

Comment:
I think I wrote `the column` to you, but I meant `this column` 🙃

Reply:
✅ Fixed